epub.d.ts

```diff
@@ -30,37 +30,63 @@
  *
  *      /images/logo_img/OPT/logo.jpg
  **/
-declare module "epub" {
-	
-	import {EventEmitter} from "events";
-	
-	interface TocElement {
-		level: number;
-		order: number;
-		title: string;
-		id: string;
-		href?: string;
-	}
-	
-	class EPub extends EventEmitter {
-		constructor(epubfile: string, imagewebroot?: string, chapterwebroot?: string);
-		
-		metadata: Object;
-		manifest: Object;
-		spine: Object;
-		flow: Array<Object>;
-		toc: Array<TocElement>;
-		
-		parse(): void;
-		
-		getChapter(chapterId: string, callback: (error: Error, text: string) => void): void;
-		
-		getChapterRaw(chapterId: string, callback: (error: Error, text: string) => void): void;
-		
-		getImage(id: string, callback: (error: Error, data: Buffer, mimeType: string) => void): void;
-		
-		getFile(id: string, callback: (error: Error, data: Buffer, mimeType: string) => void): void;
-	}
-	
-	export = EPub;
+import { EventEmitter } from 'events'
+
+declare class EPub extends EventEmitter {
```

Comment:
this was auto-formatted with [prettier](https://prettier.io/), sorry about that. It happens by default in my editor, so it happened at one point while editing. If it's a sticking point, i'm happy to go through and un-format it.